log/log.go

```diff
@@ -0,0 +1,32 @@
+package log
+
+import "fmt"
+
+type Logger struct {
+	verbose bool
+}
+
+func NewLogger(verbose bool) Logger {
+	return Logger{verbose: verbose}
+}
+
+func (l Logger) Print(a ...any) (n int, err error) {
```

Comment:
We don't use explicit returns anywhere in the code base (or shouldn't if we do) so let's avoid it here as well.